modules/EvseManager/Charger.cpp

```diff
@@ -1083,7 +1076,14 @@ bool Charger::start_transaction() {
     shared_context.transaction_active = true;
 
     const types::powermeter::TransactionReq req{
-        evse_id, shared_context.session_uuid, shared_context.id_token.id_token.value, 0, 0, ""};
+        evse_id,
+        shared_context.session_uuid,
+        true,
+        {},
+        utils::convert_to_ocmf_identification_type(shared_context.id_token.id_token.type),
+        std::nullopt,
+        shared_context.id_token.id_token.value,
+        std::nullopt};
```

Comment:
This is hard to read, might help to have comments or set each field by hand - also very error prone